roles/nginx/tasks/nginx_yum.yml

```diff
@@ -1,3 +1,11 @@
+---
+- name: Install dependencies for nginx
```

Comment:
I believe it's not dependencies for nginx, but dependencies for http://docs.ansible.com/ansible/seboolean_module.html module to work.

See: https://github.com/StackStorm/ansible-st2/pull/92/files#diff-84ef76b67429d6f947d82255e8a3a5e5R36